src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -133,6 +131,11 @@ export const OptionsListPopoverActionBar = ({
                   />
                 </EuiToolTip>
               </EuiFlexItem>
+              {!hideSort && (
+                <EuiFlexItem grow={false}>
+                  <OptionsListPopoverSortingButton showOnlySelected={showOnlySelected} />
+                </EuiFlexItem>
+              )}
```

Comment:
I love this new sort button! 🤩 

![Sep-26-2023 14-36-32](https://github.com/elastic/kibana/assets/8698078/eeecb192-b8b5-4834-9596-ad5d1a5a59d4)